samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/main.c

```diff
@@ -128,7 +128,7 @@ void update_light_state(void)
 
 void main(void)
 {
-	int err;
+	s32_t err;
```

Comment:
bt_enable() returns int and not s32_t. Why did you change this?

Reply:
Ok I will make it int again

Reply:
done !!